### DISCUSSION TITLE
How to add a protected exe to the list?

### DISCUSSION THREAD

**Discussion by bervel** [nefarius/HidHide#142] [Q&A] [answered]
Hi, did anyone manage to get around how to add a protected exe to the list of apps? MSFS purchased via Windows store would be one such example. If you try to select the exe file with the config GUI ("+" ...) you get a Windows "No permissions ..." pop-up and you're not able to add it to the list.

**Comment by nefarius** [comment]
What is a "protected exe" even? Do you mean a UWP/MS Store app? If so, those are not supported.

**Reply by bervel** [reply]
I mean MSFS for example, like I noted originally. I'm pretty sure Microsoft Flight Simulator is not an UWP app, even if I did buy it via Microsoft Store.

If I've checked the advanced security properties of FlightSimulator.exe and there is a special condition set for it for "Users" group: WIN://SYSAPPID Contains "Microsoft.FlightSimulator_8wekyb...". I think that's the thing that's blocking the selection via Hidhide GUI client. I can't remove this condition manually and even if I did, I'm affraid it might cause problems.

**Reply by nefarius** [reply]
Sounds like an UWP app to me, which is not supported.

**Reply by bervel** [reply]
Look mate, if you're just trying to get rid of me - fine. But please don't ask me to pretend that MSFS, game fo the year, 1,4 million sales on Steam, one of the most HW demanding games is some kind of a portable UWP app.

**Reply by nefarius** [reply, marked as answer]
Why would it being an UWP app imply that it is somehow on the level of a mobile app? It is not, and IDK why the sudden shift in tone was necessary. UWP applications are not lesser or inferior to traditional Win32 executables, ***however*** they run on an architecture that is not supported by HH's core design and can not be configured, end of story. That's all there is about it.

**Reply by nefarius** [reply]
Here, [took me 3 seconds on a search engine](https://forums.flightsimulator.com/t/installing-reshade-on-ms-store-version-of-the-msfs2020-via-reshadeforuwp-program-by-milkydeveloper/411855) to check that it is in fact a UWP application.
